openscm/adapters/__init__.py

```diff
@@ -33,13 +33,14 @@ def load_adapter(name: str) -> type:
             if name == "MODELNAME":  # pragma: no cover
                 from .modelname import MODELNAME
 
-                adapter = MODELNAME(parameters)
+                adapter = MODELNAME
```

Comment:
Also added a sentence to the docs clarifying that users should return their adapter class rather than an instance of that.